content/user/auth_general.php

```diff
@@ -1,10 +0,0 @@
-<?php
-	//used to authenticate users for gallery3
-  include("../../util.inc");
-   
-  if (isset($_COOKIE["cmubuggy_auth"])){
```

Comment:
Is this the "single sign on" bit? i.e. Will users still be able to log in to the gallery and forum separately with their gallery and forum specific creds?

Reply:
As far as I can tell, yes -- I don't have any main site creds and was able to log into the forum just fine.

As for the gallery, it wasn't clear to me if the logins worked at all so I didn't try? :)